protocol/avdl/keybase1/teams.avdl

```diff
@@ -248,6 +283,23 @@ protocol teams {
     boolean isDeleted;
   }
 
+  record Signer {
+    Seqno e; // eldest seqno
+    KID k;   // signer KID
+    UID u;   // signer UID
+  }
+
+  record HiddenTeamChainLink {
+    @jsonkey("m") @mpackkey("m")
+    MerkleRootV2 merkleRoot;
```

Comment:
What merkle root? Merkle root that this link is first included in? Merkle root it signed over?